telescope/src/main/java/com/mattprecious/telescope/TelescopeLayout.java

```diff
@@ -631,7 +656,7 @@ private void unregisterRequestCaptureReceiver() {
               }
             }
           }
-        }, null);
+        }, getHandler());
```

Comment:
Ugh, was relying on the compiler to help me with my last minute renaming of things... little did I know that `getHandler()` is already a method on `View`. Thanks!